src/main/resources/templates/timezones/index.html

```diff
@@ -20,6 +20,11 @@
     <h1>Timezones</h1>
 
     <p>Make your splash page here</p>
+
+    <form action="/group9/index.html" method="post">
+        <button class="btn btn-ok" type="submit">click for at gå videre</button>
```

Comment:
The link to your page should be added as a card on the templates/index.html file next to the timezone card. Follow the same format as the one present already.

Reply:
Værsgo vi har fået det rettet men index.html fungere ikke. Vi har brug for hjælp